### ISSUE TITLE
Recommend additions to qlog support

### ISSUE THREAD

**Issue by larseggert** [quiclog/qvis#20]
It would be useful if qvis, when loading a qlog file, would recommend additional fields or other qlog features for the generating application to implement, in order to improve the visualization or better support qvis features.

**Comment by rmarx** [comment]
That's a good suggestion @larseggert, though short-term I will probably mainly add more README's per visualization to explain which fields they use. 

Is there a concrete example you encountered that motivated opening this issue?

**Comment by larseggert** [comment]
Nope. I just keep wondering if I could improve my qlog support and how...
